Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -99,9 +107,11 @@ class HTTPClientTests: TestCase {
 
         let request = HTTPRequest(method: .post([:]), path: .mockPath)
 
-        self.client.perform(request) { (_: EmptyResponse) in }
+        waitUntil { completion in
+            self.client.perform(request) { (_: EmptyResponse) in completion() }
+        }
 
-        expect(headerPresent.value).toEventually(equal(true))
```

Comment:
Looks like a few of these didn't have a timeout and will have it now so it's a slight change, but I think those are ok.

Reply:
`toEventually` had a default timeout as well.